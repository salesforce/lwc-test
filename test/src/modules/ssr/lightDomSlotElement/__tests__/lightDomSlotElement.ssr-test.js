/*
 * Copyright (c) 2018, salesforce.com, inc.
 * All rights reserved.
 * SPDX-License-Identifier: MIT
 * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
 */
import { renderComponent } from 'lwc';
import LightDomSlotElement from 'ssr/lightDomSlotElement';

it('renders a basic component with light DOM slot with element slotted', () => {
    const renderedComponent = renderComponent(
        'ssr-light-dom-slot-element',
        LightDomSlotElement,
        {},
    );

    expect(renderedComponent).toMatchSnapshot();
});
