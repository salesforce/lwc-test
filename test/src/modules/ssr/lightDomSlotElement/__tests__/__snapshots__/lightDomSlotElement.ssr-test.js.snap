// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a basic component with light DOM slot with element slotted 1`] = `
<ssr-light-dom-slot-element>
  <ssr-light-dom-slottable>
    <div>
      Hello
    </div>
  </ssr-light-dom-slottable>
</ssr-light-dom-slot-element>
`;
